src/somevars.js

```diff
@@ -0,0 +1 @@
+export const isloggedin = false;

```

Comment:
Change the name of the file from somevars.js to something meaningful like constants.js or actionTypes.js